.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
```

Comment:
@rmotitsuki feel free to nuke `.travis.yml`, GH Actions have demonstrated a great alternative based on the credits given for kytos-ng org (this is mapped in another issue btw issue #10)